disco_aws_automation/socify_helper.py

```diff
@@ -110,7 +119,7 @@ def _invoke_socify(self, function_name, status=None, **kwargs):
         data = self._build_json(status, **kwargs)
         logger.debug("calling Socify with data : %s", data)
         headers = {'Content-Type': 'application/json'}
-        return requests.post(url=url, headers=headers, json=data)
+        return self.request_session.post(url=url, headers=headers, json=data, timeout=10)
```

Comment:
Double-check if we need to clean up request_session eventually

Reply:
Also, use a variable/constant for 10

Reply:
Check default timeout and set it to higher than 30 sec if the default is less than a min

Reply:
Looks like by default if we do not specify a timeout, it will not time out and can hang forever. So I will set the timeout to 60 seconds.